Formula/fortio.rb

```diff
@@ -0,0 +1,39 @@
+class Fortio < Formula
+  desc "HTTP and gRPC load testing and visualization tool and server"
+  homepage "https://github.com/istio/fortio"
+  url "https://github.com/istio/fortio/archive/v0.9.0.tar.gz"
+  sha256 "76871124e7abd13d7fa0ccc5ad6912bc1c3d77d53694da2b26ad44af3638edab"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["XC_OS"] = "darwin"
+    ENV["XC_ARCH"] = "amd64"
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
+    contents = Dir["{*,.git,.gitignore}"]
```

Comment:
what happens if you just use `buildpath.children`?

Reply:
No issues with it